### ISSUE TITLE
[Bug][a11y] TalkBack doesn't announce the search bar being in focus when you tap Start browsing

### ISSUE THREAD

**Issue by sv-ohorvath** [mozilla-mobile/fenix#7851]
## Steps to reproduce
1. Install Fenix. Turn TalkBack on.
2. On the onboarding screen, tap Start Browsing.
3. Check what TB reads on the search screen.
### Expected behavior
When you open the search bar, TB should read the search bar placeholder text.
### Actual behavior
TB only announces the keyboard is on, but not the screen/item focused on.
### Device information

* Android device: Pixel 3 (Android 10), Galaxy Note 8 (Android 9)
* Fenix version: 3.2-beta1

**Comment by sblatz** [comment]
For more clarity here: the focus is not brought to the search bar when going from Start Browsing Button -> Search screen but it **is** brought focus from normal home screen to search screen.
